docs/configuration.md

```diff
@@ -10,7 +9,7 @@ To make this a cinch, Lychee utilizes the [DotEnv][1] PHP library by Vance Lucas
 
 Your `.env` file should not be committed to your application's source control, since each developer / server using your application could require a different environment configuration. Furthermore, this would be a security risk in the event an intruder gains access to your source control repository, since any sensitive credentials would get exposed.
 
-If you are developing with a team, you may wish to continue including a .env.example file with your application. By putting placeholder values in the example configuration file, other developers on your team can clearly see which environment variables are needed to run your application. You may also create a .env.testing file. This file will override the .env file when running PHPUnit tests or executing Artisan commands with the --env=testing option.
+If you are developing with a team, you may wish to continue including an `.env.example` file with your application. By putting placeholder values in the example configuration file, other developers on your team can clearly see which environment variables are needed to run your application. You may also create an `.env.testing` file. This file will override the `.env` file when running PHPUnit tests or executing Artisan commands with the `--env=testing` option.
```

Comment:
I agree with the ``s. I'm curious about the group opinion on "a" vs "an" with a leading dot. I read them as "dot env dot example", so would use "a".